src/mono/wasm/debugger/tests/debugger-test-chinese-char-in-path-ㄨ/debugger-test-chinese-char-in-path-ㄨ.csproj

```diff
@@ -0,0 +1,5 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PublishWindowsPdb>false</PublishWindowsPdb>
```

Comment:
Why is this needed?

Reply:
Without this we get an error on CI:

`The command ""C:\Users\itomkowicz\source\repos\runtime-fork\.packages\microsoft.diasymreader.pdb2pdb\1.1.0-beta2-19575-01\tools\Pdb2Pdb.exe" "C:\Users\itomkowicz\source\repos\runtime-fork\artifacts\bin\debugger-test\Debug\wasm\debugger-test-special-char-in-path.dll" /out "C:\Users\itomkowicz\source\repos\runtime-fork\artifacts\SymStore\Debug\debugger-test-special-char-in-path\net6.0\x64\debugger-test-special-char-in-path.pdb" /srcsvrvar SRC_INDEX=public" exited with code -1.`

Reply:
Any idea what would cause this? And details before that line?

Reply:
Pdb2Pdb.exe does not support special characteres in path, as far as I remember.

Reply:
Ok, we can add a comment for that here. And if there is any supporting link with more info, then add that.